src/ray/ray_exported_symbols.lds

```diff
@@ -27,3 +28,8 @@
 *init_raylet*
 *Java*
 *JNI_*
+#boost
+*boost*;
```

Comment:
Why we need boost symbols?

Reply:
boost symbols shouldn't be exported. And we should try to avoid export symbols as we can. Symbols confilic is tricky and hard to debug

Reply:
removed, it's only used when compiled in debug mode.